### PR TITLE
honey sandwich

### PR DESCRIPTION
**Pull request by Cyrano7**
Thanks to @dbackeus for the idea. Adds a honey sandwich, removes honey components from sandwich_sauce.

### CODE REVIEW

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -12145,6 +12142,32 @@
...
+    [
+      [ "flatbread", 2 ],
+      [ "bread", 2 ],
+      [ "cornbread", 2 ],
+      [ "wastebread", 2 ]
+    ],
+    [
+      [ "honeycomb", 1 ],
+      [ "honey_bottled", 1 ]
+    ]
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_food.json]: Why no "honey_glassed"?

**Comment by Cyrano7** [comment]
Is crystallized honey spreadable? I guess it depends on how crystallized it is. I left it out because I figured it would be not suitable for sandwich making.

**Comment by Coolthulhu** [comment]
I don't think it's crystallized so hard that you can't spread it with a knife. Most crystallized honey I've seen was just annoying to deal with, not unsuitable for sandwiches.

**Comment by Cyrano7** [comment]
@Coolthulhu I suppose so. Its kinda after the fact but i'm 50/50 on it so if you really want it I can double back and add it.

**Pull request merged**